assets/js/components/tree.js

```diff
@@ -63,6 +63,7 @@ class TreeCollection {
         this._tree         = tree;
         this._el           = $(el);
         this._toggle       = this._el.find('> [data-role="toggle"]');
+        this._toggleBtn    = this._toggle.find('button');
```

Comment:
i think it's weird that the actual clickable element is the h4, not the button. it would make more sense to move the `data-role` attribute to the button and use the button's focus etc.

Reply:
Makes sense - ended up removing the h4 entirely to simplify styling